oap-server/server-receiver-plugin/zipkin-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/zipkin/ZipkinReceiverConfig.java

```diff
@@ -33,4 +36,5 @@
     private long jettyIdleTimeOut = 30000;
     private int jettyAcceptorPriorityDelta = 0;
     private int jettyAcceptQueueSize = 0;
+    private List<String> instanceNameRule = new ArrayList<>();
```

Comment:
Do OAP core support putting list type?

Reply:
support

Reply:
Could you add a unit test to verify? I can't find the test about this for now. And existing codes use string, and then convert to list.

Reply:
@klboke I can't see the change, why do you resolve this?

Reply:
I am going to add a test case. Is there a similar test case for reference?

Reply:
https://github.com/apache/skywalking/pull/6763/commits/0245b00cd3303b6222de9a2d6bd5206986e41c3b

Reply:
Test case to prove the property with list type working with the current framework.

Reply:
I didn’t understand what you mean, is there any example you can refer to?

Reply:
Take a look at `ModuleManagerTest`.